app/controllers/organisations_controller.rb

```diff
@@ -0,0 +1,59 @@
+class OrganisationsController < ApplicationController
+  attr_reader :content_item
```

Comment:
What's the reason to add the `attr_reader` here?

Reply:
This was to display the content item from the content store on the page for stubbing purposes but I believe this can be removed

Reply:
That's not a good idea to add code for testing purposes.
I believe you can remove it and it will still work

Reply:
Yes, I will remove it